app/helpers/findings_helper.rb

```diff
@@ -413,4 +413,18 @@ def summary_for review
 
       "#{ConclusionReview.human_attribute_name 'summary'}: #{summary}"
     end
+
+    def finding_issues_amount finding
+      finding.issues.sum &:amount
+    end
+
+    def amount_impact finding
+      amount = finding_issues_amount finding
+
+      Finding::AMOUNT_IMPACT.detect { |name, import| amount <= import }
+    end
+
+    def finding_impact_risks_type finding
+      finding.impact_risk_text(amount_impact(finding).first)
```

Comment:
Esto debería verse así:
```suggestion
      finding.impact_risk_text
```